### PR TITLE
FP-699: Prototype: setup static pages

### PR DESCRIPTION
**Pull request by m-asim-iqbal**
Please review this PR:
@dynamicmonkeys @radicalgeek

### CODE REVIEW

**Review by radicalgeek — APPROVED** [review]

File: app/views/500.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+{% block pageTitle %}
+  GOV.UK 500 page template – {{ serviceName }} – GOV.UK Prototype Kit
+{% endblock %}
+
+{% block content %}
+ <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <h1 class="govuk-heading-l">500 Server Error  </h1>
+            <p class="govuk-body">
+            Oops,something went wrong at {{" - " +requestId if requestId else "" }}
```
Comment (dynamicmonkeys) [inline app/views/500.html]: Given this is just a prototype, does it need this?

File: app/views/accessibility.html [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+{% block content %}
+ <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full">
+            <h1 class="govuk-heading-xl">
+                Accessibility statement for the Find Transport Data service
+            </h1>
+
+            <p class="govuk-body-m">
+                This statement only applies to the Find Transport Data service, available at <a href="https://findtransportdata.dft.gov.uk">https://findtransportdata.dft.gov.uk</a>.
```
Comment (dynamicmonkeys) [inline app/views/accessibility.html]: Can you please knock this href to being the root of the prototype please

File: app/views/components/footer.njk [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/footer/macro.njk" import govukFooter %}
+
+{{ govukFooter({
+  classes:"ftd-footer",
```
Comment (dynamicmonkeys) [inline app/views/components/footer.njk]: I can't see where we leverage this as a JS hook nor have a custom style for it, if we don't, can you please remove it so we can nurture this to be less custom, less ftd-* and nap-*, TY

File: app/views/404.html [hunk 4]
```diff
@@ -0,0 +1,23 @@
+{% extends "layouts/main.html" %}
+
+{% block pageTitle %}
+  GOV.UK 404 page template – {{ serviceName }} – GOV.UK Prototype Kit
```
Comment (dynamicmonkeys) [inline app/views/404.html]: Please align the pageTitle with the prod one, wrapping in serviceName as appropriate

File: app/views/500.html [hunk 5]
```diff
@@ -0,0 +1,23 @@
+{% extends "layouts/main.html" %}
+
+{% block pageTitle %}
+  GOV.UK 500 page template – {{ serviceName }} – GOV.UK Prototype Kit
```
Comment (dynamicmonkeys) [inline app/views/500.html]: Please align the pageTitle with the prod one, wrapping in serviceName as appropriate

File: app/views/accessibility.html [hunk 6]
```diff
@@ -0,0 +1,110 @@
+{% extends "layouts/main.html" %}
+
+{% block pageTitle %}
+  GOV.UK accessibility template – {{ serviceName }} – GOV.UK Prototype Kit
```
Comment (dynamicmonkeys) [inline app/views/accessibility.html]: Please align the pageTitle with the prod one, wrapping in serviceName as appropriate

File: app/views/cookies.html [hunk 7]
```diff
@@ -0,0 +1,165 @@
+{% extends "layouts/main.html" %}
+
+{% block pageTitle %}
+  GOV.UK cookies page template – {{ serviceName }} – GOV.UK Prototype Kit
```
Comment (dynamicmonkeys) [inline app/views/cookies.html]: Please align the pageTitle with the prod one, wrapping in serviceName as appropriate

File: app/views/cookies.html [hunk 8]
```diff
@@ -0,0 +1,165 @@
...
+            <div id="cookie-settings__no-js">
+                <h2 class="govuk-heading-s govuk-!-margin-top-6">Do you want to accept analytics cookies?</h2>
+                <p class="govuk-body">We use Javascript to set our analytics cookies. Unfortunately Javascript is not running on your browser, so you cannot change your settings. You can try:</p>
+                <ul class="govuk-list govuk-list--bullet">
+                <li>reloading the page</li>
+                <li>turning on Javascript in your browser</li>
+                </ul>
+            </div>
+            <div class="cookie-settings__form-wrapper">
+                <form data-module="cookie-settings" action="javascript:saveCookieChoice()">
```
Comment (dynamicmonkeys) [inline app/views/cookies.html]: Does saveCookieChoice hook up?

File: app/views/homepage.html [hunk 9]
```diff
@@ -0,0 +1,97 @@
+{% extends "layouts/main.html" %}
+
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/phase-banner/macro.njk" import govukPhaseBanner %}
+
+{% block pageTitle %}
+  GOV.UK homepage template – {{ serviceName }} – GOV.UK Prototype Kit
```
Comment (dynamicmonkeys) [inline app/views/homepage.html]: Please align the pageTitle with the prod one, wrapping in serviceName as appropriate

File: app/views/homepage.html [hunk 10]
```diff
@@ -0,0 +1,97 @@
...
+{% endblock %}
+
+{% block content %}
+<div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <span class="govuk-caption-l ftd-small-heading">Find transport data</span>
```
Comment (dynamicmonkeys) [inline app/views/homepage.html:14]: .ftd-small-heading: The sass for this is missing :)

File: app/views/layouts/main.html [hunk 11]
```diff
@@ -4,3 +4,99 @@
...
+  {{ govukButton({
+    type:"submit",
+    classes:"govuk-!-margin-bottom-0 ",
+    html: searchIcon
+  }) }}
+</div>
+{% endset %}
+
+{% set searchHtml %}
+<form action="/browse/" method="get">
```
Comment (dynamicmonkeys) [inline app/views/layouts/main.html]: /browse/ should be /browse so Google et al don't see it as a separate page, if it's like this in the main repo then a ticket needs raising to mop up

File: app/views/privacy.html [hunk 12]
```diff
@@ -0,0 +1,68 @@
+{% extends "layouts/main.html" %}
+
+{% block pageTitle %}
+  GOV.UK privacy page template – {{ serviceName }} – GOV.UK Prototype Kit
```
Comment (dynamicmonkeys) [inline app/views/privacy.html]: Please align title

File: app/views/roadmap.html [hunk 13]
```diff
@@ -0,0 +1,48 @@
+{% from "govuk/components/phase-banner/macro.njk" import govukPhaseBanner %}
+{% extends "layouts/main.html" %}
+
+{% block pageTitle %}
+  GOV.UK roadmap page template – {{ serviceName }} – GOV.UK Prototype Kit
```
Comment (dynamicmonkeys) [inline app/views/roadmap.html]: Please align pageTitle

File: app/views/roadmap.html [hunk 14]
```diff
@@ -0,0 +1,48 @@
...
+{% endblock %}
+
+{{ govukPhaseBanner({
+  tag: {
+    text: "alpha"
+  },
+  html: 'This is a new service – your <a class="govuk-link" href="#">feedback</a> will help us to improve it.'
```
Comment (dynamicmonkeys) [inline app/views/roadmap.html]: If this # href is right, then this needs mopping up on the main repo

File: app/views/terms-and-conditions.html [hunk 15]
```diff
@@ -0,0 +1,256 @@
+{% extends "layouts/main.html" %}
+
+{% block pageTitle %}
+  GOV.UK term and conditions page template – {{ serviceName }} – GOV.UK Prototype Kit
```
Comment (dynamicmonkeys) [inline app/views/terms-and-conditions.html]: Please align

**Review by dynamicmonkeys — CHANGES_REQUESTED** [review]
@m-asim-iqbal good work, very helpful. A handful of tweaks please for titles etc to align

File: app/views/500.html (hunk 1 above)
Comment (m-asim-iqbal) [reply app/views/500.html]: Changed to static string

File: app/views/accessibility.html (hunk 2 above)
Comment (m-asim-iqbal) [reply app/views/accessibility.html]: Changed the href to "/"

File: app/views/components/footer.njk (hunk 3 above)
Comment (m-asim-iqbal) [reply app/views/components/footer.njk]: Removed classes

File: app/views/404.html (hunk 4 above)
Comment (m-asim-iqbal) [reply app/views/404.html]: Changed page title to "GOV.UK - Find transport data"

File: app/views/500.html (hunk 5 above)
Comment (m-asim-iqbal) [reply app/views/500.html]: Changed page title to "GOV.UK - Find transport data"

File: app/views/layouts/main.html (hunk 11 above)
Comment (m-asim-iqbal) [reply app/views/layouts/main.html]: Updated

File: app/views/privacy.html (hunk 12 above)
Comment (m-asim-iqbal) [reply app/views/privacy.html]: Changed page title to "GOV.UK - Find transport data"

File: app/views/roadmap.html (hunk 13 above)
Comment (m-asim-iqbal) [reply app/views/roadmap.html]: Changed page title to "GOV.UK - Find transport data"

File: app/views/roadmap.html (hunk 14 above)
Comment (m-asim-iqbal) [reply app/views/roadmap.html]: updated href to smartsurvey url

File: app/views/terms-and-conditions.html (hunk 15 above)
Comment (m-asim-iqbal) [reply app/views/terms-and-conditions.html]: Updated

**Pull request merged**